lib/queue/worker.py

```diff
@@ -64,7 +68,7 @@ def safely_respond(self, model: Model) -> List[schemas.Message]:
         if success:
             self.delete_processed_messages(messages_with_queues)
```

Comment:
I think a metric to count successes?  (with an attribute for model type, so we can know how much is sent to each one?)

Reply:
will add!